pom.xml

```diff
@@ -47,6 +47,7 @@
         <iotdevicesdk.version>1.10.3</iotdevicesdk.version>
         <picocli.version>4.6.1</picocli.version>
         <dagger.version>2.37</dagger.version>
+        <apache.commons.version>3.7</apache.commons.version>
```

Comment:
Do not bring all of apache commons lang just to test for "countMatches". Write the code yourself.

Reply:
removed in d98e9434740bb90c6989c794c9d58e006903b9e1